tests/api/conftest.py

```diff
@@ -11,153 +24,120 @@ def patch_request(module_mocker):
 @pytest.fixture(scope='module', autouse=True)
 def patch_check_env(module_mocker):
     module_mocker.patch(
-        'dataall.api.Objects.Environment.resolvers.check_environment',
+        'dataall.core.environment.api.resolvers.check_environment',
         return_value='CDKROLENAME',
     )
     module_mocker.patch(
-        'dataall.api.Objects.Environment.resolvers.get_pivot_role_as_part_of_environment', return_value=False
+        'dataall.core.environment.api.resolvers.get_pivot_role_as_part_of_environment', return_value=False
     )
 
 
 @pytest.fixture(scope='module', autouse=True)
 def patch_es(module_mocker):
-    module_mocker.patch('dataall.searchproxy.connect', return_value={})
-    module_mocker.patch('dataall.searchproxy.search', return_value={})
-    module_mocker.patch('dataall.searchproxy.base_indexer.BaseIndexer.delete_doc', return_value={})
+    module_mocker.patch('dataall.base.searchproxy.connect', return_value={})
+    module_mocker.patch('dataall.base.searchproxy.search', return_value={})
+    module_mocker.patch('dataall.core.catalog.indexers.base_indexer.BaseIndexer.delete_doc', return_value={})
 
 
 @pytest.fixture(scope='module', autouse=True)
 def patch_stack_tasks(module_mocker):
     module_mocker.patch(
-        'dataall.aws.handlers.ecs.Ecs.is_task_running',
+        'dataall.core.stacks.aws.ecs.Ecs.is_task_running',
         return_value=False,
     )
     module_mocker.patch(
-        'dataall.aws.handlers.ecs.Ecs.run_cdkproxy_task',
+        'dataall.core.stacks.aws.ecs.Ecs.run_cdkproxy_task',
         return_value='arn:aws:eu-west-1:xxxxxxxx:ecs:task/1222222222',
     )
     module_mocker.patch(
-        'dataall.aws.handlers.cloudformation.CloudFormation.describe_stack_resources',
+        'dataall.core.stacks.aws.cloudformation.CloudFormation.describe_stack_resources',
         return_value=True,
     )
 
 
 @pytest.fixture(scope='module', autouse=True)
 def permissions(db):
     with db.scoped_session() as session:
-        yield dataall.db.api.Permission.init_permissions(session)
+        yield Permission.init_permissions(session)
 
 
 @pytest.fixture(scope='module', autouse=True)
 def user(db):
-    with db.scoped_session() as session:
```

Comment:
saving the user to db was unnecessary? then db fixture not needed as function input